### DISCUSSION TITLE
dD spatial range searching with point with info

### DISCUSSION THREAD

**Discussion by wongshek** [CGAL/cgal#5964] [Q&A] [answered]
I have seen many examples about K neighbor searching with point with info, but I didn't find an example of range search with point with info. Is it possible to implement this kind of query?

**Comment by sloriot** [comment, marked as answer]
There is no example but some tests. See for example:
https://github.com/CGAL/cgal/blob/master/Spatial_searching/test/Spatial_searching/Range_searching.cpp

**Reply by wongshek** [reply]
Thank you!
